app/models/policy_host.rb

```diff
@@ -7,4 +7,10 @@ class PolicyHost < ApplicationRecord
 
   validates :policy, presence: true
   validates :host, presence: true, uniqueness: { scope: :policy }
+
+  def self.import_from_policy(policy_id, host_ids)
+    PolicyHost.import(host_ids.map do |host_id|
```

Comment:
`import` is in the local scope:

```suggestion
    import(host_ids.map do |host_id|
```